tests/e2e-inst-test.sh

```diff
@@ -8,13 +8,50 @@ ARCH=$(uname -m)
 
 TRACEE_STARTUP_TIMEOUT=30
 TRACEE_SHUTDOWN_TIMEOUT=30
-TRACEE_RUN_TIMEOUT=60
 SCRIPT_TMP_DIR=/tmp
 TRACEE_TMP_DIR=/tmp/tracee
 
 # Default test to run if no other is given
 TESTS=${INSTTESTS:=VFS_WRITE CONTAINERS_DATA_SOURCE WRITABLE_DATA_SOURCE DNS_DATA_SOURCE PROCTREE_DATA_SOURCE PROCESS_EXECUTE_FAILED LSM_TEST}
 
+TRACEE_POLICY_PATH="./tests/policies/inst/"
+
+# Test configuration: policy_name:timeout:internal_sleep
+declare -A TEST_CONFIG_MAP=(
+    ["SET_FS_PWD"]="set-fs-pwd-test:5:0"                             # set_fs_pwd.sh: timeout 5s, no sleep
+    ["WRITABLE_DATA_SOURCE"]="writable-ds-test:40:0"                 # writable_data_source.sh: timeout 40s, no sleep
+    ["SECURITY_PATH_NOTIFY"]="security-path-notify-test:5:0"         # security_path_notify.sh: timeout 5s, no sleep
+    ["SUSPICIOUS_SYSCALL_SOURCE"]="suspicious-syscall-src-test:10:0" # suspicious_syscall_source.sh: timeout 10s, no sleep
+    ["CONTAINERS_DATA_SOURCE"]="containers-ds-test:10:5"             # containers_data_source.sh: timeout 10s, container sleeps 5s
+    ["PROCTREE_DATA_SOURCE"]="proctree-ds-test:20:15"                # proctree_data_source.sh: timeout 20s, proctreetester program sleeps 15s
+    ["HOOKED_SYSCALL"]="hooked-syscall-test:10:5"                    # hooked_syscall.sh: timeout 10s, script sleeps 5s
+    ["PROCESS_EXECUTE_FAILED"]="execute-failed-test:5:2"             # process_execute_failed.sh: timeout 5s, script sleeps 2s
+    ["STACK_PIVOT"]="stack-pivot-test:10:5"                          # stack_pivot.sh: timeout 10s, stack_pivot program sleeps 5s
+    ["FTRACE_HOOK"]="ftrace-hook-test:15:5"                          # ftrace_hook.sh: timeout 15s, script sleeps 5s
+    ["BPF_ATTACH"]="bpf-attach-test:15:5"                            # bpf_attach.sh: timeout 15s, script sleeps 5s
+    ["DNS_DATA_SOURCE"]="dns-ds-test:10:0"                           # dns_data_source.sh: timeout 10s, no sleep
+    ["SECURITY_INODE_RENAME"]="security-inode-rename-test:10:2"      # security_inode_rename.sh: timeout 10s, script sleeps 2s
+    ["FILE_MODIFICATION"]="file-modification-test:5:0"               # file_modification.sh: timeout 5s, no sleep
+    ["LSM_TEST"]="lsm-test:5:0"                                      # lsm_test.sh: timeout 5s, no sleep
+    ["VFS_WRITE"]="vfs-write-test:5:0"                               # vfs_writev.sh: timeout 5s, no sleep
+)
+
+# Helper functions to extract config values
+get_policy_name() {
+    local test_name="$1"
+    echo "${TEST_CONFIG_MAP[$test_name]}" | cut -d: -f1
+}
+
+get_test_timeout() {
+    local test_name="$1"
+    echo "${TEST_CONFIG_MAP[$test_name]}" | cut -d: -f2
+}
+
+get_test_sleep() {
+    local test_name="$1"
+    echo "${TEST_CONFIG_MAP[$test_name]}" | cut -d: -f3
+}
```

Comment:
Move common logic into test common script to be sourced by this, kernel and net tests.